nifi-toolkit/nifi-toolkit-assembly/pom.xml

```diff
@@ -308,5 +308,135 @@ language governing permissions and limitations under the License. -->
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>rpm</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>unpack-shared-resources</id>
+                                <goals>
+                                    <goal>unpack-dependencies</goal>
+                                </goals>
+                                <phase>generate-resources</phase>
+                                <configuration>
+                                    <outputDirectory>${project.build.directory}/generated-resources</outputDirectory>
+                                    <includeArtifactIds>nifi-resources</includeArtifactIds>
+                                    <includeGroupIds>org.apache.nifi</includeGroupIds>
+                                    <excludeTransitive>false</excludeTransitive>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>unpack-docs</id>
+                                <goals>
+                                    <goal>unpack-dependencies</goal>
+                                </goals>
+                                <phase>generate-resources</phase>
+                                <configuration>
+                                    <outputDirectory>${project.build.directory}/generated-docs</outputDirectory>
+                                    <includeArtifactIds>nifi-docs</includeArtifactIds>
+                                    <includeGroupIds>org.apache.nifi</includeGroupIds>
+                                    <excludeTransitive>false</excludeTransitive>
+                                    <excludes>LICENSE,NOTICE</excludes>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>rpm-maven-plugin</artifactId>
+                        <configuration>
+                            <name>nifi-toolkit</name>
+                            <summary>Apache NiFi Toolkit</summary>
+                            <description>Apache NiFi Toolkit</description>
```

Comment:
The description should be more detailed like [nifi-registry](https://github.com/apache/nifi/blob/2daac5714a6050dd872522ff74d7fe71f700db6e/nifi-registry/nifi-registry-assembly/pom.xml#L283).

My suggestion:
```
The NiFi Toolkit contains several command line utilities to setup and support NiFi in standalone and clustered environments.
```
from [Apache NiFi Toolkit Guide](https://nifi.apache.org/docs/nifi-docs/html/toolkit-guide.html).